x11-packages/xcape/build.sh

```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/alols/xcape
+TERMUX_PKG_DESCRIPTION="Linux utility to configure modifier keys to act as other keys when pressed and released on their own."
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.2
+TERMUX_PKG_SRCURL=https://github.com/alols/xcape/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=a27ed884fd94f03058af65a39edfe3af3f2f8fbb76ba9920002a76be07fb2821
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_BUILD_DEPENDS="libx11,libxtst"
```

Comment:
Neither of the dependencies are in the X11 repo, so please put this package in the main `packages` repo as well, unless there is an unstated reason this package depends on a graphical output.

Reply:
those are build dependencies

Reply:
And that means the package needs to go in the X11 repo?

Reply:
it needs x11.

```
$ DISPLAY= xcape
Unable to connect to X11 display. Is $DISPLAY set?
```

Reply:
Ah okay, thanks for the confirmation.